streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -82,16 +85,16 @@ public boolean test(String key, String value) {
                 }
             });
 
-        KStream<String, Integer> stream2 = stream1.mapValues(new ValueMapper<String, Integer>() {
+        KStream<String, Integer> stream2 = stream1.mapValues(new ValueMapperWithKey<String, String, Integer>() {
```

Comment:
we need tests with both `ValueMapper` and `ValueMapperWithKey`. Same applies to other methods that have been added